packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.tsx

```diff
@@ -24,6 +26,7 @@ const WalletDropdown: React.FC<TProps> = ({
     label,
     list,
     listHeight = 'md',
+    maxWidth = '19.5rem',
```

Comment:
But what if we don't need a limit, like with the full-width dropdown on mobile (e.g. transactions tab)? Do we pass `100%` / `undefined` / `none` / `unset`? Maybe make default `maxWidth` dependent on `isMobile`?

`const dropdownMaxWidth = maxWidth || isMobile ? 'none' : '19.5rem';` and remove default prop assignment?

Reply:
The idea behind using maxWidth instead of width is to make it flexible and not only tied to one specific width. By setting the `width = 100%` and passing the value for maxWidth it will take the full width of the container for responsive (if responsive width is less than the set maxWidth for desktop) or if width should be less than width of container then we can just limit it via maxWidth.

If no limit and a value higher than the default width of 19.5rem is needed we can pass the width of the container or 100%.

Reply:
In the context of the Transactions page dropdown filter we can add something like this to the `WalletDropdown` component:
`maxWidth={isMobile ? '100%' : '19.5rem'}`

Let me know your opinion on this solution